### DISCUSSION TITLE
Problem with dynamic import (I think) after upgrade from v2.14.0 to latest

### DISCUSSION THREAD

**Discussion by adirpion** [software-mansion/react-native-reanimated#4609] [Q&A]
Hi,
I'm using react-native 0.70.9 in my project and testerday I upgraded from v2.14.0 to the latest version (3.3.0).
I run build in my android and after my build succeed, I run my app with metro by "yarn start" and I got this error:
<img width="890" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/53223169/bca16588-4fe8-433d-a2e4-90e7cf12f848">

This problem because of dynamic imports I think, but I don't know why reanimated library crash it.
Can you give me some solution for me this problem ?

Thanks :)

**Comment by tjzel** [comment]
Have you tried resetting metro's cache? `yarn start --reset-cache`

**Reply by adirpion** [reply]
Hi,
Thanks for you reply.
I tried to do it but still I got this error.

**Reply by tjzel** [reply]
I don't recall any issues related to dynamic imports with reanimated. Could you maybe open an issue and provide some repro?
